README.md

```diff
@@ -55,3 +55,5 @@ curl -vv \
 
 # Debugging
 Visit https://notifications-staging.zooniverse.org/ for debug user & testing section announcements and user notifications.
+
+[![pullreminders](https://pullreminders.com/badge.svg)](https://pullreminders.com?ref=badge)

```

Comment:
Can this be first thing at the top of the page above the first heading?

Reply:
Done